src/vs/workbench/contrib/terminal/browser/xterm/xtermTerminal.ts

```diff
@@ -143,12 +143,6 @@ export class XtermTerminal extends Disposable implements IXtermTerminal, IDetach
 
 	private readonly _onDidRequestRunCommand = this._register(new Emitter<{ command: ITerminalCommand; copyAsHtml?: boolean; noNewLine?: boolean }>());
 	readonly onDidRequestRunCommand = this._onDidRequestRunCommand.event;
-	private readonly _onDidRequestFocus = this._register(new Emitter<void>());
-	readonly onDidRequestFocus = this._onDidRequestFocus.event;
-	private readonly _onDidRequestSendText = this._register(new Emitter<string>());
-	readonly onDidRequestSendText = this._onDidRequestSendText.event;
-	private readonly _onDidRequestFreePort = this._register(new Emitter<string>());
-	readonly onDidRequestFreePort = this._onDidRequestFreePort.event;
```

Comment:
cleanup 🧹

Reply:
Hi, is this PR going to be released in the next VSCode release?